guide/event-handling/README.md

```diff
@@ -0,0 +1,144 @@
+# Event handling
+
+Node.js uses an event-driven architecture. This makes it possible to execute code when a certain event occurs. The discord.js library takes full advantage of this. You can visit the [discord.js documentation site](https://discord.js.org/#/docs/main/stable/class/Client) to see the full list of `Client` events.
+
+Here's the base code we'll be using:
+
+```js
+const Discord = require('discord.js');
+const { token } = require('./config.json');
+
+const client = new Discord.Client();
+
+client.once('ready', () => {
+	console.log('Ready!');
+});
+
+client.on('message', message => {
+	console.log(`${message.author.tag} in #${message.channel.name} sent: ${message.content}`);
+});
+
+client.login(token);
+```
+
+Currently, the event listeners are placed in the `index.js` file. The `ready` event emits once when the `Client` becomes ready for use and the `message` event emits whenever a message is received. Moving the event listener code into individual files is simple and we'll be taking a similar approach to the [command handler](/command-handling/). 
+
+## Individual event files
+
+Your project structure should look something like this:
+
+![Project structure before events folder](./images/project-structure-before-events-folder.png)
+
+Create an `events` folder in the directory where your `index.js` file is kept. This is how your project structure should look after that:
+
+![Project structure after events folder](./images/project-structure-after-events-folder.png)
+
+Now, you'll take your existing events code in `index.js` and move them to individual files inside the `events` folders. Create a `ready.js` and a `message.js` file in the `events` folder and place in the code for the respective files:
```

Comment:
That image is missing the `commands` folder (not important here, but it will be consistent with the previous section) and it also looks a little different from these two.

Reply:
That's the idea; there is no commands folder in this guide section and the image used on this page should look the same as the existing one.

Reply:
Yeah, I realized that after posting that comment. Was on my way to commit the changes. 😓